src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -436,6 +436,7 @@ public void testVersion() throws Exception {
     final Version version = sut.version();
     assertThat(version.apiVersion(), not(isEmptyOrNullString()));
     assertThat(version.arch(), not(isEmptyOrNullString()));
+    assertThat(version.buildTime(), not(isEmptyOrNullString()));
```

Comment:
Since `buildTime` was added in API 1.22, this assertion will fail when tested against earlier APIs. Wrap this assertion in an `if(versionAtLeast("1.22"))` condition. (I don't remember exactly what the helper method to test the version is called, but you can find other examples around the test class.)